php/EE/Runner.php

```diff
@@ -778,6 +791,23 @@ private function auto_check_update() {
 		exit;
 	}
 
+	/**
+	 * Triggers migration if current phar version > version in ee_option table
+	 */
+	private function maybe_trigger_migration() {
+		$db_version = Option::get( 'version' );
```

Comment:
Run phpcbf. Equal to sign should match the lower one.

Reply:
The reason I didn't catch this was because we don't have a rule in PHPCS that checks for alignment of = sign (and marks it as error). The tool that we use to fix phpcs errors is a phpstrom code formatter that also does additional code cleanings as aligning = signs.

Reply:
Please update the editor configurations accordingly.